client/editSchedule.js

```diff
@@ -261,6 +313,27 @@ Template.editSchedule.rendered = function() {
   setupCompetitionCalendar(template, $calendar, $addEditSomethingModal);
 };
 
+Template.unscheduledRound.rendered = function() {
+  var template = this;
+  var $unscheduledRound = template.$('.fc-event');
+
+  // store data so the calendar knows to render an event upon drop
+  // $unscheduledRound.data('event');
+
+  // make the event draggable using jQuery UI
+  $unscheduledRound.draggable({
+    zIndex: 999,
+    revert: false,      // will cause the event to go back to its
```

Comment:
Oh, that comment was a 2 line comment, it got removed because the option above it got deleted. I'll remove the comment. While Blaze automatically removes the item from the external palette, it will actually flicker as it goes back and then gets removed by Blaze (if revert is set to true).